docs/design/analytics.md

```diff
@@ -46,7 +46,7 @@ Note: There's a limit of 20 custom dimensions.
 | 4 | `Node Version` | `number` |
 | 5 | `Flag: --style` | `string` |
 | 6 | `--collection` | `string` |
-| 7 | `--buildEventLog` | `boolean` |
+| 7 | `Flag: --strict` | `boolean` |
```

Comment:
@kyliau, can you please check what changes need to be done in GA to rename the dimension?

Reply:
GA only cares about the ID - 7 in this case. I've renamed Dimension ID 7 to `flag: --strict`